src/js/remoteServices.js

```diff
@@ -158,11 +158,9 @@ export default {
     name: "trello",
     host: "https://trello.com",
     urlPatterns: [":host:/c/:id/:title"],
-    description: (document, service, { title }) =>
-      document.querySelector("#js-dialog-title")?.textContent?.trim() || title,
-    projectId: (document) =>
-      projectIdentifierBySelector("#js-dialog-title")(document) ||
-      projectIdentifierBySelector("[data-testid='board-name-display']")(document),
+    description: (document, _service, { title }) =>
+      document.querySelector("#card-back-name")?.textContent?.trim() || title,
+    projectId: (document) => projectIdentifierBySelector("#card-back-name")(document),
```

Comment:
The project identifier must be read from the board name, thus this must be the selector containing the board name. E.g. we have Sprint [175] as board name, 175 is the identifier of the project that should be pre-selected for the board.

Reply:
i see, but this selector doesn't work anymore

Reply:
will check which will work

Reply:
![SCR-20241002-itop-3](https://github.com/user-attachments/assets/28560e2b-be4b-4ccd-afb6-cabf5e7e7bbf)

`[data-testid="board-display-name"]` should work